netlify.toml

```diff
@@ -4,3 +4,9 @@
 
 [context.production]
     environment = { JEKYLL_ENV = "production" }
+
+[[redirects]]
+  from = "/assets/Cal-ITP.MobilityMarketplace.DataPlans.NASPO.FirstNet.pdf"
+  to = "/how-to/get-connected"
+  status = 301
+  force = false
```

Comment:
Tried adding this here, based on Netlify docs here: https://docs.netlify.com/configure-builds/file-based-configuration/#redirects and https://docs.netlify.com/routing/redirects/redirect-options/#query-parameters

But it's not working on the deploy preview - not sure why: https://deploy-preview-410--cal-itp-mobility-marketplace.netlify.app/assets/Cal-ITP.MobilityMarketplace.DataPlans.NASPO.FirstNet.pdf

Reply:
`force = true` fixed it! ❇️ ❇️ 

try [https://deploy-preview-410--cal-itp-mobility-marketplace.netlify.app/how-to/get-connected now](https://deploy-preview-410--cal-itp-mobility-marketplace.netlify.app/assets/Cal-ITP.MobilityMarketplace.DataPlans.NASPO.FirstNet.pdf) now

Reply:
Yea @angela-tran this way with the `netlify.toml` file works better in this case with a PDF file. There's no way to add code to a PDF file.

Reply:
Oh I see -- totally missed that. Makes sense!